hcf_backend/__init__.py

```diff
@@ -273,8 +273,20 @@ def schedule(self, batch):
         self.logger.info('scheduled %d links' % scheduled)
 
     def _process_hcf_link(self, link, score):
-        link.meta.pop(b'origin_is_frontier', None)
-        hcf_request = {'fp': getattr(link, 'meta', {}).get('hcf_fingerprint', link.url)}
+        link_meta = getattr(link, 'meta', {})
+        link_meta.pop(b'origin_is_frontier', None)
+        scrapy_meta = link_meta.get(b'scrapy_meta', {})
+        hcf_request = (
+            link_meta.get('hcf_request')
+            or scrapy_meta.get('hcf_request')
```

Comment:
To pass custom ``qdata`` or ``fdata`` values. I don't have a concrete example for this as the priority handling already does what I need.

Either way, for my use case I need one of:
* translate scrapy priority to hcf priority transparently
* allow to pass somehow hcf's ``p`` parameter